### PR TITLE
Pre verify sudo authorization "-v" before calling curl.

### PR DESCRIPTION
**Pull request by MCT-MIT-RCP**
The way this was, the curl interferes with the
	sudo and bash invocation. Here it will authenticate first.

#### What is the purpose of this change?

Documentation "correction" showing the sequence of "verify sudo" then "use sudo" before any other commands would interfere with the input/output

#### Was the change discussed in an issue or in the forum before?

No.

#### Checklist

- [ N ] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-new-feature-or-bug-fix).
- [ N/A ] I have added tests for all changes in this PR if appropriate.
- [ N/A ] I have added documentation for the changes if appropriate.
- [ ?" ] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [ N ] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

File: docs/content/install.md [hunk 1]
```diff
@@ -26,11 +26,11 @@ using the [rclone selfupdate](/commands/rclone_selfupdate/) command.
 
 To install rclone on Linux/macOS/BSD systems, run:
 
-    curl https://rclone.org/install.sh | sudo bash
+    sudo -v ; curl https://rclone.org/install.sh | sudo bash
```
Comment (ncw) [inline docs/content/install.md:29]: Maybe this should be

    sudo -v && curl ...

So we don't run the curl etc if the sudo fails?

Comment (ncw) [reply docs/content/install.md:29]: I'm happy to defer to your experience here - I'm not a shell scripting expert.

So you recommend we merge as-is with the `;` and experiment after?

Comment (MCT-MIT-RCP) [reply docs/content/install.md:29]: Yes, exactly. I also suspect that the behavior of `sudo &&` across platforms may not be consistent enough to rely on.

Random thought if there is a "Would be nice" task list:
"Create a test script that looks for some of the corner cases of sudo in this shell 'phrase'"

Comment (ncw) [reply docs/content/install.md:29]: OK I'll merge what we;ve got so far - thank you.

Doing cross platform experiments with shell scripts is tricky indeed since I don't have access to all the platforms we support!
